src/helpers/projectsList.js

```diff
@@ -1,63 +0,0 @@
-import project01 from "./../img/projects/01.jpg"
-import project01Big from "./../img/projects/01-big.jpg"
-
-import project02 from "./../img/projects/02.jpg"
-import project02Big from "./../img/projects/02-big.jpg"
-
-import project03 from "./../img/projects/03.jpg"
-import project03Big from "./../img/projects/03-big.jpg"
-
-import project04 from "./../img/projects/04.jpg"
-import project04Big from "./../img/projects/04-big.jpg"
-
-import project05 from "./../img/projects/05.jpg"
-import project05Big from "./../img/projects/05-big.jpg"
-
-import project06 from "./../img/projects/06.jpg"
-import project06Big from "./../img/projects/06-big.jpg"
-
-const projects =[
-    {
```

Comment:
this file is not a helper responsibility, but just data needed for some component, please place it in the same folder where u use it